starter-code/src/functions-and-arrays.js

```diff
@@ -48,6 +106,15 @@ var wordsUnique = [
   'bring'
 ];
 
+function uniquifyArray (palabras){
+  let resultado = palabras.filter(function(item, index){
+	  return palabras.indexOf(item) >= index;
+  });
+  return resultado
```

Comment:
Mejor devuelve directamente el resultado de la función `filter`. Me llama la atención que hayas usado `for` antes y ahora uses `filter`